lib/caps/MTL/info

```diff
@@ -0,0 +1,13 @@
+###
+### Copyright (C) 2021 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+###
+### kate: syntax python;
+###
+
+info = dict(
+  gpu = dict(gen = 12.2),
```

Comment:
Is this the correct gpu gen?  @FocusLuo can you confirm?

Reply:
updated